src/openapi.json

```diff
@@ -252,6 +257,11 @@
 						"type": "string",
 						"example": "Example Inc."
 					},
+					"keycloakOrganizationId": {
+						"description": "If available, the PDC Keycloak Organization ID for this changemaker.",
+						"type": "string",
+						"format": "uuid"
+					},
```

Comment:
Does this need `nullable: true`?

Since we have a few copies of this now, is it worth extracting a type to use as a `$ref`?

Reply:
Done, nice suggestion!